commands/utils/docker-endpoint.ts

```diff
@@ -9,14 +9,8 @@ class DockerClient {
     private endPoint:Docker;
 
     constructor() {
-        if (process.env.DOCKER_HOST){
-            this.endPoint = new Docker();
-        }
-        else if (process.platform === 'win32') {
-            this.endPoint = new Docker({ socketPath: "//./pipe/docker_engine" });
-        } else {
-            this.endPoint = new Docker({ socketPath: '/var/run/docker.sock' });
-        }
+        // Pass no options so that the defaultOpts of docker-modem will be used
+        this.endPoint = new Docker(null);
```

Comment:
@tasosval Ah good call. Why don't we just update our local copy of the typings file for now? Then we can discuss pushing the improvements back to a shared copy. Ideally, the `dockerode` lib would ship with its d.ts file via NPM.